frappe/public/js/frappe/list/list_renderer.js

```diff
@@ -233,25 +233,13 @@ frappe.views.ListRenderer = Class.extend({
 	},
 	add_column: function (df) {
 		// field width
-		var colspan = 3;
-		if (in_list(['Int', 'Percent'], df.fieldtype)) {
-			colspan = 2;
-		} else if (in_list(['Check', 'Image'], df.fieldtype)) {
-			colspan = 1;
-		} else if (in_list(['name', 'subject', 'title'], df.fieldname)) {
-			// subjects are longer
-			colspan = 4;
-		} else if (df.fieldtype == 'Text Editor' || df.fieldtype == 'Text') {
-			colspan = 4;
-		}
-
```

Comment:
Apologies, I will open up a separate PR with this commit removed. Would you like me to open an issue relating to this?